LSPDFR+/EnhancedTrafficStop.cs

```diff
@@ -104,6 +106,16 @@ public EnhancedTrafficStop()
         {
             if (Functions.IsPlayerPerformingPullover())
             {
+
+                if (!HasShownKeybindHelp)
+                {
+                    string maybeControllerButton = BringUpTrafficStopMenuControllerButton != ControllerButtons.None ? $" or ~o~{BringUpTrafficStopMenuControllerButton}~w~" : "";
+
+                    Game.DisplayNotification($"Press ~b~{BringUpTrafficStopMenuKey.FriendlyName()}~w~{maybeControllerButton} when standing at the side of the vehicle to bring up the ~g~LSPDFR+ Traffic Stop~w~ menu instead of the normal Traffic Stop menu.");
```

Comment:
.FriendlyName() cannot be found anywhere

Reply:
I forgot to make a pull request for this:

https://github.com/Albo1125/Albo1125-Common/pull/3

Also, in my testing so far, I'm not sure that the LSPDFR+ tactics do anything, possibly because setting pursuit task flags will have no effect if 0.4 cops aren't actually being given pursuit tasks...